libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/xpub.getAddress.integration.test.ts

```diff
@@ -1,12 +1,11 @@
+import coininfo from "coininfo";
+
 import { DerivationModes } from "..";
 import BitcoinLikeStorage from "../storage";
 import BitcoinLikeExplorer from "../explorer";
 import Xpub from "../xpub";
-import coininfo from "coininfo";
 import BCHCrypto from "../crypto/bitcoincash";
 import BTCCrypto from "../crypto/bitcoin";
-import ZECCrypto from "../crypto/zec";
-import ZENCrypto from "../crypto/zen";
 
```

Comment:
why the zec and zen tests are removed?

Reply:
I think it is OK, it seems that you've moved the zec and zen tests to another file. Right?

Reply:
Yes it's just moved